keychain/tests/test_keychain.py

```diff
@@ -21,6 +21,7 @@ def setUp(self):
 
         self.keychain = self.env['keychain.account']
         config['keychain_key'] = Fernet.generate_key()
+        config['running_env'] = ""
```

Comment:
It makes the line 26 useless.
Instead : 
`self.old_running_env = config.get('running_env', '')`